llvm-pretty-bc-parser.cabal

```diff
@@ -135,7 +135,7 @@ Test-suite disasm-test
                        tasty-hunit,
                        tasty-sugar >= 2.2 && < 2.3,
                        text,
-                       versions,
+                       versions < 6,
```

Comment:
Feel free to undo this, as I've just released `6.0.1`.

Reply:
Thanks, @fosskers!